lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -67,53 +71,59 @@ const AvailableDays = styled(FieldSet)`
     }
   }
   & > span {
-    border: 1px solid ${BORDER_COLOR};
-    border-left: none;
+    align-items: center;
+    border-radius: 3rem;
     box-sizing: border-box;
-    display: inline-block;
-    height: 3em;
-    max-width: 150px;
-    min-width: 14.28%;
+    display: inline-flex;
+    flex-direction: row-reverse;
+    height: 3rem;
+    min-width: 4.5rem;
```

Comment:
Can we adjust the width/padding on these to make them more like pills? They're looking a little squished as is.

Reply:
How do the changes from https://github.com/opentripplanner/otp-react-redux/pull/1238/commits/8d8715f342c67205bb797ff3263227575670b3a1 look like?

Reply:
Looks good thank you! I think the margin could be adjusted to something like `0 7px 0 2px` to look a little more even?

Reply:
Adjusted in https://github.com/opentripplanner/otp-react-redux/pull/1238/commits/7395d62977c3509feb45137950acbb25fe5c1cdd